.travis.yml

```diff
@@ -0,0 +1,13 @@
+---
+cache: pip
+language: python
+
+python:
+  - "3.7"
+  - "3.7-dev"  # 3.7 development branch
+
+install:
+- pip install .[all]
+
+script:
+- pytest
```

Comment:
I think this file is missing the newline at the end

Reply:
done